src/js/lib/yasr.bundled.js

```diff
@@ -55640,7 +55640,9 @@ var root = module.exports = function(parent, options, queryResults) {
 		}
 	}
 
-
+    yasr.isExplainPlan = function() {
+        return window.editor.getValue().includes('onto:explain') || window.editor.getValue().includes('http://www.ontotext.com/explain');
```

Comment:
We can move this check directly in highlightExplainPlan() in query-editor-directive.js. so that yasr do not need to know about window.editor.getValue and explain plan details.